packages/SwingSet/src/lib/workerOptions.js

```diff
@@ -10,6 +10,8 @@ export async function makeWorkerOptions(managerType, bundleHandler) {
   } else if (managerType === 'xsnap' || managerType === 'xs-worker') {
     const bundleIDs = await bundleHandler.getCurrentBundleIDs();
     return harden({ type: 'xsnap', bundleIDs });
+  } else if (managerType === 'node-subprocess') {
+    return harden({ type: 'node-subprocess' });
```

Comment:
Take a look at the Node command line options doc https://nodejs.org/api/cli.html and see if you'd want any of these being baked into the (semi-)permanent definition of a vat.  *I'm* pretty convinced the answer is no.

Reply:
In discussion, @warner and I noticed that Node vats can only resume execution after a stop by performing a replay from the very beginning, which leaves us without any actual use case for `nodeOptions` plasticity -- you always have to start over from the beginning no matter what.  This strengthens the argument for making `nodeOptions` sticky, since, all other things being equal, it's cleaner to have a uniform mechanism for handling managerType-specific configuration information regardless of the actual managerType.  On that basis we've decided to go with the sticky option.

If at some point Node acquires the ability to capture and restart from memory snapshots the way XS can, we might have to revisit this choice.  But if the inconvenience of having to go back and change things in such an eventuality acts as a sort of applied-Murphyism to cause such a feature appear, we'd be happy enough with that as the outcome.